docs/synthetic-tokens/what-are-synthetic-assets.md

```diff
@@ -45,15 +46,25 @@ By using UMA's priceless synthetic token contract template, developers can easil
 
 
-### The ExpiringMultiParty (EMP) Contract Template 
+### The ExpiringMultiParty (EMP) Contract Template
 
-One can write priceless financial contract templates to create various kinds of financial products.The first type of synthetic contracts on UMA  is used for creating expiring synthetic tokens and is called the ExpiringMultiParty (EMP) contract. 
+One can write priceless financial contract templates to create various kinds of financial products. The first type of synthetic contracts on UMA is used for creating expiring synthetic tokens and is called the ExpiringMultiParty (EMP) contract.
 
-The EMP allows token sponsors (i.e., people who mint new synthetic tokens) to collateralize their position in a defined collateral currency. Anytime after the EMP’s expiration date, any synthetic tokenholder can redeem their tokens for a settlement value, which is denominated in the collateral currency and fixed to the price of the EMP contract’s price identifier at the expiration timestamp. The price determining the settlement value is resolved by UMA’s oracle contract, the “DVM”. 
+The EMP allows token sponsors (i.e., people who mint new synthetic tokens) to collateralize their position in a defined collateral currency. Anytime after the EMP’s expiration date, any synthetic tokenholder can redeem their tokens for a settlement value, which is denominated in the collateral currency and fixed to the price of the EMP contract’s price identifier at the expiration timestamp. The price determining the settlement value is resolved by UMA’s oracle contract, the “DVM”.
 
 This is only the first example of a synthetic token implementation, and is by no means restrictive on the types of synthetic tokens that could be built on UMA's infrastructure.
 
-View [here](synthetic-tokens/expiring-synthetic-tokens.md) to learn more about UMA's EMP contract. 
+View [here](synthetic-tokens/expiring-synthetic-tokens.md) to learn more about UMA's EMP contract.
+
+### The Perpetual Contract Template
+
+Another contract template approved for use on UMA is the perpetual, which can be used to create synthetic tokens that follow the upward and downward price movement of any asset without an expiration date.
+
+Like the EMP, the perpetual allows token sponsors to collateralize their position in a defined collateral currency and track the price of a different asset through a defined price feed. The synthetic token can be redeemed by token sponsors at any time for its value in the underlying collateral, and the synthetic token's value relative to the underlying collateral is nudged up or down through a funding rate mechanism. By changing the value of the synthetic, the funding rate mechanism effectively pays the token sponsor a fee when the price being tracked goes down, and pays the token holder a fee when the price being tracked goes up.
```

Comment:
> By changing the value of the synthetic, the funding rate mechanism effectively pays the token sponsor a fee when the price being tracked goes down, and pays the token holder a fee when the price being tracked goes up.

I don't think this is completely accurate. Really the funding rate pays a token sponsor a fee when the price of the perpetual is trading at a premium to the underlying index and a fee to the token holder when the perp is trading at a discount.

Also think that it is important to also convey that the purpose of the funding rate is to keep the token pegged to movements in the underlying.